src/main/resources/static/pages/popups/phraseAddress.html

```diff
@@ -3,7 +3,7 @@
         <div class="modal-content">
             <div class="modal-header">
                 <h4 class="modal-title">Generate address with mnemonic phrase</h4>
-                <span>Be sure to remember phrase and write it on paper. Private key will not be kept on server.</span>
+                <span>Mnemonic phrase is less secure than private key, but it's easy to remember. Phrase/private key will not be kept on server.</span>
```

Comment:
why is a Mnemonic phrase less secure than a private key?

Reply:
It's about 128 bits of entropy in it while private key is 256 bit.

Reply:
is this for 12 word mnemonics?

Reply:
@ligi we use 5 words + 1 pseudo-word
12 words is less than 256 bit too but has more entropy than our approach
it's a compromise between ability to remember and strength
the thing you couldn't remember you will write on the paper or store on disk, both of which are security threats too